### PR TITLE
Updates publications page

### PR DESCRIPTION
**Pull request by mrniaboc**

### CODE REVIEW

File: app/pages/about/publications-page.cjsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ publicationCategories =
       {citation: "Planet Hunters. VIII. Characterization of 41 Long-Period Exoplanet Candidates from Kepler Archival Data, Wang+ 2015."
       href: "http://arxiv.org/pdf/1512.02559v2.pdf"},
       {citation: "Planet Hunters. IX. KIC 8462852 – Where’s the flux?, Boyajian+ 2015."
-      href: "http://mnras.oxfordjournals.org/content/457/4/3988.full.pdf"},
+      href: "http://arxiv.org/pdf/1509.03622v1.pdf"},
```
Comment (willettk) [inline app/pages/about/publications-page.cjsx]: Any reason this is going back to the arXiv version, @mrniaboc? Where possible, we should try to link to the actual manuscript version if it's open access.

**Comment by srallen** [comment]
Would addressing #1850 help with @willettk's question?

**Comment by willettk** [comment]
Definitely, @srallen. I don't know why it's being reverted for this particular case, as I thought all articles in this journal should be open source. #1850 is a great idea, though.

**Comment by srallen** [comment]
@mrniaboc is it ok if I close this for now and fold this into completing #1850?

**Comment by srallen** [comment]
Closing in favor of #2653

**Pull request closed without merging**